src/qlik-date-picker.js

```diff
@@ -48,6 +48,29 @@ define(["qlik", "jquery", "./lib/moment.min", "./calendar-settings", "css!./lib/
             });
             return dateStates;
         }
+        /** Encodes input for use in HTML context
```

Comment:
This looks like it is just copy/pasted code. What is the source? Formatting seems a bit off as well.

Reply:
Updated the code to include the license needed for Encoder.js